darts/explainability/tft_explainer.py

```diff
@@ -0,0 +1,249 @@
+from typing import Dict, List, Literal, Optional
+
+import matplotlib.pyplot as plt
+import pandas as pd
+from torch import Tensor
+
+from darts import TimeSeries
+from darts.explainability.explainability import (
+    ExplainabilityResult,
+    ForecastingModelExplainer,
+)
+from darts.models import TFTModel
+
+
+class TFTExplainer(ForecastingModelExplainer):
+    def __init__(
+        self,
+        model: TFTModel,
+    ):
+        """
+        Explainer class for the TFT model.
+
+        Parameters
+        ----------
+        model
+            The fitted TFT model to be explained.
+        """
+        super().__init__(model)
+
+        if not model._fit_called:
+            raise ValueError("The model needs to be trained before explaining it.")
+
+        self._model = model
+
+    @property
+    def encoder_importance(self):
```

Comment:
Could you add docstrings explaining what this and `decoder_importance` are returning? They can be quite useful I think.

Reply:
I have added docstrings to the module and properties. I wasn't 100% sure on the details of the model so if you could have a look at it that would be great? If everything is fine you can resolve this conversation.

Reply:
Hey, I was trying to call this function and ran into an error. it says no attribute 'encoder_sparse_weights'. I went to the tft_model.py and uncommented this code chunk : 
return self.to_network_output(
             prediction=self.transform_output(out, target_scale=x["target_scale"]),
             attention=attn_out_weights,
             static_variables=static_covariate_var,
             encoder_variables=encoder_sparse_weights,
             decoder_variables=decoder_sparse_weights,
             decoder_lengths=decoder_lengths,
             encoder_lengths=encoder_lengths,
         )

It now says TFTModule has no attribute called to_network_output.

Can I get some help regarding how to call the explainer and use it in my code?